src/intl/messages.js

```diff
@@ -516,13 +520,16 @@ export const messages: { [messageId: string]: MessageType } = {
   pendingChanges: 'Pending Changes',
   pleaseEnterValidHostName: "Please enter a valid host name. Only lower-case and upper-case letters, numbers, and '_','-','.' are allowed.",
   pleaseEnterValidVmName: "Please enter a valid virtual machine name. Only lower-case and upper-case letters, numbers, and '_','-','.' are allowed.",
+  preferredConsole: 'Preferred Console',
```

Comment:
I would change it to "console type"...

Reply:
Providing a correct naming gets tricky here... just "console type" could confuse users as this is not a console that will be used for **every** VM. It will be the first choice of course but console needs to be supported by the VM.

Reply:
Right, but "Preferred Console" is also tricky from the same reason. That's why we might need a tooltip for explaining it.